addons/web_unsplash/models/ir_qweb.py

```diff
@@ -8,6 +8,8 @@ class Image(models.AbstractModel):
 
     @api.model
     def from_html(self, model, field, element):
+        if element.find('.//img') is None:
```

Comment:
Why is the condition not checking the same xpath as the one used after?
IMO
```python
if element.find('.//img') is None:
```
Should target same element:
```python
if element.find('img') is None:
```

Reply:
It is due to copy paste from master branch I created previously ;)